solutions/kubernetes/cluster-setup.sh

```diff
@@ -63,13 +63,6 @@ function setup_cluster() {
         # Join worker nodes.
         JOIN_WORKER="/var/tmp/cluster-functions.sh --join-worker-nodes $JOIN_COMMAND"
         pdsh -w ^/var/tmp/pdsh-hosts "$JOIN_WORKER"
-
-        # Label worker nodes.
```

Comment:
why we removed code from this file?

Reply:
answered in https://github.com/Seagate/cortx-re/pull/1156#discussion_r939865692